<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundSky</class>
 <widget class="QFrame" name="SoundSky">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SOUND SKY - MUSIC</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchBar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Search...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="searchSongsBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/searchmagnifierinterfacesymbol1_79893.ico</normaloff>../../Downloads/searchmagnifierinterfacesymbol1_79893.ico</iconset>
   </property>
  </widget>
  <widget class="QComboBox" name="filterSearch">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Track</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Genre</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Artist</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="songsContainer">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>391</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
